eng/Versions.props

```diff
@@ -53,6 +53,11 @@
     <MicrosoftBuildLocatorVersion>1.2.6</MicrosoftBuildLocatorVersion>
     <MicrosoftBuildRuntimeVersion>15.3.409</MicrosoftBuildRuntimeVersion>
     <MicrosoftBuildTasksCoreVersion>15.3.409</MicrosoftBuildTasksCoreVersion>
+    
+    <NuGetVisualStudioContractsVersion>5.7.0-rtm.6702</NuGetVisualStudioContractsVersion>
```

Comment:
FYI, we just published 5.7.0 (no pre-release label), in case you want to use it. https://www.nuget.org/packages/NuGet.VisualStudio.Contracts/5.7.0

Reply:
good to know!